luigi/tools/range.py

```diff
@@ -402,6 +402,84 @@ def _format_datetime(self, dt):
         return luigi.DateHourParameter().serialize(dt)
 
 
+class RangeByMinutesBase(RangeBase):
+    """
+    Produces a contiguous completed range of an recurring tasks separated a specified number of minutes.
+    """
+    start = luigi.DateMinuteParameter(
+        default=None,
+        description="beginning date-hour-minute, inclusive. Default: None - work backward forever (requires reverse=True)")
+    stop = luigi.DateMinuteParameter(
+        default=None,
+        description="ending date-hour-minute, exclusive. Default: None - work forward forever")
+    minutes_back = luigi.IntParameter(
+        default=60*24,  # one day
+        description=("extent to which contiguousness is to be assured into "
+                     "past, in minutes from current time. Prevents infinite "
+                     "loop when start is none. If the dataset has limited "
+                     "retention (i.e. old outputs get removed), this should "
+                     "be set shorter to that, too, to prevent the oldest "
+                     "outputs flapping. Increase freely if you intend to "
+                     "process old dates - worker's memory is the limit"))
+    minutes_forward = luigi.IntParameter(
+        default=0,
+        description="extent to which contiguousness is to be assured into future, "
+                    "in minutes from current time. Prevents infinite loop when stop is none")
+
+    minutes_interval = luigi.IntParameter(
+        default=5,
```

Comment:
@Tarrasch, thanks for reviewing it. 
Not sure what you mean with your first comment, is it just the choice of name of RangeByMinutes vs RangeMinutely? English is not my mother tongue, but [minutely ](http://english.stackexchange.com/questions/3091/weekly-daily-hourly-minutely)didn't sound correct to me.

About the default value of 5 minutes interval. Ultimately it responds to the fact it is the interval that was needed in our project. I could justify it as being a reasonable interval in the minutes range, making every minute a too frequent interval... but it is not trouble to change it.